scripts/missions/toau/08_A_Mercenary_Life.lua

```diff
@@ -32,8 +32,10 @@ mission.sections =
 
     {
         check = function(player, currentMission, missionStatus, vars)
-            return currentMission == mission.missionId and
-                not mission:getMustZone(player)
+            return
+                currentMission == mission.missionId
+                and os.time() >= player:getCharVar('Mission[4][7]Stage')
```

Comment:
Ah! So now that we're in A_Mercenary_Life.lua, we can simply go os.time() >= mission:getVar(player, 'Stage')